components/eam/src/chemistry/utils/modal_aero_wateruptake.F90

```diff
@@ -23,7 +23,9 @@ module modal_aero_wateruptake
 
 public :: &
    modal_aero_wateruptake_init, &
-   modal_aero_wateruptake_dr
+   modal_aero_wateruptake_dr, &
+   modal_aero_kohler
```

Comment:
I may have missed something but I don't see modal_aero_kohler being used anywhere outside of this module.

Reply:
It's used for ECPP, see src/physics/crm/ecpp/ecpp_modal_wetscav.F90